microsoft-edge/webview2/concepts/distribution.md

```diff
@@ -41,6 +41,12 @@ Cons:
 *  Your WebView2 app cannot specify that a particular version of the WebView2 Runtime is required.
 
 
+<!-- ------------------------------ -->
+<!-- #### Which devices already have WebView2 Runtime installed and how to distribute for edge cases -->
+
+The Evergreen Runtime is preinstalled onto all Windows 11 devices as a part of the Windows 11 operating system.  Microsoft installed the WebView2 Runtime to all _eligible_ Windows 10 devices, as described in [Delivering Microsoft Edge WebView2 Runtime to managed Windows 10 devices](https://blogs.windows.com/msedgedev/2022/12/14/delivering-microsoft-edge-webview2-runtime-to-managed-windows-10-devices/).  Even if your app uses the Evergreen distribution mode, we recommend that you distribute the WebView2 Runtime, to cover edge cases where the Runtime wasn't already installed.  See [Details about the Evergreen Runtime distribution mode](#details-about-the-evergreen-runtime-distribution-mode) below.
```

Comment:
"installed the WebView2 Runtime to all _eligible_ Windows 10 devices"
does that contradict blog wording which is:
Runtime is now installed on **most active eligible** Windows 10 Consumer devices and will continue to be deployed on **newly active** Windows 10 devices